### PR TITLE
Resource library integration

### PR DESCRIPTION
**Pull request by manojSRawat**

### CODE REVIEW

File: src/app/client/src/app/modules/sourcing/components/chapter-list/chapter-list.component.ts [hunk 1]
```diff
@@ -1634,4 +1649,141 @@ export class ChapterListComponent implements OnInit, OnChanges, OnDestroy, After
...
+          children: {
+            Question: [
+              'Multiple Choice Question',
+              'Subjective Question'
+            ]
+          },
+          contentPolicyUrl: '/term-of-use.html'
+        }
+      },
+      searchFormConfig: [
```
Comment (pallakartheekreddy) [inline src/app/client/src/app/modules/sourcing/components/chapter-list/chapter-list.component.ts:1703]: Why is this Config hardcoded? shouldn't it come from API?

File: src/app/client/src/app/modules/sourcing/components/chapter-list/chapter-list.component.ts [hunk 2]
```diff
@@ -1634,4 +1649,141 @@ export class ChapterListComponent implements OnInit, OnChanges, OnDestroy, After
...
+          objectType: 'QuestionSet',
+          primaryCategory: 'Practice Question Set',
+          isRoot: true,
+          iconClass: 'fa fa-book',
+          showAddCollaborator: false,
+          children: {
+            Question: [
+              'Multiple Choice Question',
+              'Subjective Question'
+            ]
```
Comment (pallakartheekreddy) [inline src/app/client/src/app/modules/sourcing/components/chapter-list/chapter-list.component.ts:1698]: Why is this Config required for a library?

**Review by pallakartheekreddy — CHANGES_REQUESTED** [review]

File: src/app/client/src/app/modules/sourcing/components/chapter-list/chapter-list.component.ts (hunk 2 above)
Comment (manojSRawat) [reply src/app/client/src/app/modules/sourcing/components/chapter-list/chapter-list.component.ts:1698]: on the basis of objectType, we are getting the detail of questionSet/collection in the library.

Comment (pallakartheekreddy) [reply src/app/client/src/app/modules/sourcing/components/chapter-list/chapter-list.component.ts]: @manojSRawat if the library already making apical why Config hardcoded?

**Review by pallakartheekreddy — APPROVED** [review]

**Pull request merged**